style_editor.html

```diff
@@ -8830,6 +8878,42 @@
 var use_graflex = true;
 var bad_fps = 0;
 
+</script>
+
+<div id="popup_window" class="popup-window">
+  <p id="popup_message"></p>
+  <button title="OK" onclick="dismissPopupMessage()">OK</button>
+  <br>
+  <label title="Disable this popup again. Refresh page to show again.">
+    <input type="checkbox" id="dont_show_again" name="dontshow" onclick="DontShowAgain()">
+    <span class="dont-show-again-label">Don't show this again</span>
+  </label>
+</div>
+<div id="overlay" class="overlay"></div>
+
+<script>
+
+  var enablePopup = true;
+  var popupWindow = FIND("popup_window");
+  var overlay = FIND("overlay");
+
+  function showPopupMessage(message) {
+    console.log("Popup triggered. hidden?");
+    if (enablePopup) {
+      FIND("popup_message").innerHTML = message;
+      popupWindow.classList.add("show");
+      overlay.classList.add("show");
+    }
+  }
+  function dismissPopupMessage() {
+    popupWindow.classList.remove("show");
+    overlay.classList.remove("show");
+  }
+  function DontShowAgain() {
+    enablePopup = false;
+    // saveCookie("allowPopup", enablePopup);
```

Comment:
Why commented out?
Also, if there is ever another type of poup, maybe  we need to save the "don't show again" separately for each type?

Reply:
Commented as a placeholder for the saveCookie function that is yet to come. removed for now.
I'll work on adding a popupIdentifier or something.

Reply:
Seems like we should just wait until we have saveCookie?

Reply:
yes, there are other things to fix too. I'll resubmit after settings.